e2e/live/end-to-end-test.sh

```diff
@@ -481,7 +481,7 @@ printResult
 echo "Testing init for Kptfile/ResourceGroup"
 echo "kpt live init e2e/live/testdata/rg-test-case-1a"
 cp -f e2e/live/testdata/Kptfile e2e/live/testdata/rg-test-case-1a
-${BIN_DIR}/kpt live init e2e/live/testdata/rg-test-case-1a > $OUTPUT_DIR/status
+${BIN_DIR}/kpt live init e2e/live/testdata/rg-test-case-1a > $OUTPUT_DIR/status 2>&1
```

Comment:
With this change, it seems we end up with live commands sending output to a combination of stdout and stderr. This should be fixed consistently across the full kpt UX.

Reply:
Does setting [this](https://github.com/GoogleContainerTools/kpt/blob/master/commands/livecmd.go#L57) to os.StdErr is good enough to direct all logs to stderr ?

Reply:
No, the IOStreams is not used consistently across all commands. I actually think we should try to phase out the use of IOStreams and instead rely on getting the input/output streams from cobra. But I haven't had time to fix that yet.

Reply:
Can we have fixing live commands output as a follow up item ? We need this in for fn commands which is of high priority.

Reply:
Yeah, we can create a separate issue for m3 for fixing this for live.

Reply:
https://github.com/GoogleContainerTools/kpt/issues/2191